packages/playground/website-deployment/custom-redirects-lib.php

```diff
@@ -195,21 +200,21 @@ function playground_maybe_redirect( $requested_path ) {
 		str_ends_with( $requested_path, '/builder/index.php' )
 	) {
 		return array(
-			'location' => 'https://playground.wordpress.net/builder/builder.html',
+			'location' => 'builder.html',
```

Comment:
Indirectly related fix to allow testing with and redirecting to a staging site instead of playground.wordpress.net.